src/pages/settings/Subscription/CardSection/utils.ts

```diff
@@ -1,9 +1,140 @@
+import {fromUnixTime} from 'date-fns';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import DateUtils from '@libs/DateUtils';
+import type {Phrase, PhraseParameters} from '@libs/Localize';
 import * as SubscriptionUtils from '@libs/SubscriptionUtils';
+import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
+import type {Fund} from '@src/types/onyx';
+import type IconAsset from '@src/types/utils/IconAsset';
+
+type BillingStatusResult = {
+    title?: string;
+    subtitle?: string;
+    isError?: boolean;
+    isTrialActive?: boolean;
+    isRetryAvailable?: boolean;
+    isAddButtonDark?: boolean;
```

Comment:
Sorry but this variable name doesn't clearly signify the context.

Reply:
Can you specify what do you mean exactly? Or what would you suggest instead?

Reply:
I am not aware what this prop does. Neither I can't see this used in `CardSection`. Hence my question.